blueocean-dashboard/src/main/js/util/serverBrowserTimeHarmonize.js

```diff
@@ -0,0 +1,64 @@
+import moment from 'moment';
+import { logging } from '@jenkins-cd/blueocean-core-js';
+
+const logger = logging.logger('io.jenkins.blueocean.dashboard.harmonizeTimes');
+/**
+ * we need to make sure that we calculate with the correct time offset
+ *
+ * @param run
+ * @param skewMillis
+ * @returns {{ durationMillis: diff, endTime: Date, startTime: Date}}
+ */
+export class TimeManager {
+    currentTime() {
+        return moment();
+    }
+
+    harmonizeTimes(run, skewMillis) {
```

Comment:
Is the param named correctly here? I'd have thought it should be named something other than `run`. Any oject containing `startTime`, `endTime` an `durationMillis`, no?